### PR TITLE
Update copy for December changes.

### PR DESCRIPTION
**Pull request by harryharrold**
See email from Richard.

### CODE REVIEW

File: src/assessment.handlebars [hunk 1]
```diff
@@ -621,6 +621,7 @@
         <li>Contribution-based, which is based on you having paid enough National Insurance contributions</li>
       </ul>
       <p>The type of ESA you get will depend on a range of factors that we cover at the end of this guide. </p>
+      <p>In some parts of the country you can no longer make a new claim for income- related ESA, instead you have to claim Universal Credit. Eventually, Universal Credit will replace income-related ESA everywhere. This guide does not provide details about Universal Credit.</p>
```
Comment (katjam) [inline src/assessment.handlebars]: Extra space between income- and related.

Comment (katjam) [inline src/assessment.handlebars]: Extra space between income- and related at first occurence.

File: src/assessment.handlebars [hunk 2]
```diff
@@ -991,10 +992,11 @@
       <p><strong>Income-related ESA</strong> is paid to people whose income (looking at their income and their partners&#39;) and level of savings is low enough. What you receive is decided by the Government working out how much you need to live on, and paying you ESA to make up your income to that amount. In some circumstances income-related ESA may also be used to top up your contribution-based ESA.</p>
       <p>You cannot claim income-related ESA if you have savings of over &#xA3;16,000, or your partner works more than 24 hours per week.</p>
       <p>You can claim income-related ESA without time limit, although you may of course have your entitlement to ESA reassessed at some point.</p>
+      <p>In some parts of the country you can no longer make a new claim for income- related ESA, instead you claim Universal Credit. Eventually, Universal Credit will replace income-related ESA everywhere. This guide does not provide details about Universal Credit which you usually have to claim on-line. One of the places you can find out more <a href="https://www.gov.uk/universal-credit">information about Universal Credit is this page on Gov.UK.</a></p>
```
Comment (katjam) [inline src/assessment.handlebars]: Extra space between income- and related at first occurrence.

Comment (katjam) [reply src/assessment.handlebars]: Move the final fullstop outside the </a> ?

File: src/assessment.handlebars [hunk 3]
```diff
@@ -1041,8 +1043,9 @@
   <div class="slide main-menu" id="main-menu">
     <div class="flow">
       <h1 tabindex="-1">ESA Assessment</h1>
-      <p>If you can&#39;t work because of illness or disability, you could be eligible for Employment and Support Allowance &#40;<abbr title="Employment And Support Allowance">ESA</abbr>&#41;.</p>
+      <p>If you can&#39;t work because of illness or disability, you could be eligible for Employment and Support Allowance &#40;<abbr title="Employment And Support Allowance">ESA</abbr>&#41;.</p> 
       <p>We understand that the benefits system can be confusing, so this site is here to help you prepare for the assessment.</p>
+      <p>In some areas Universal Credit is replacing new claims for ESA, for more information about universal credit visit https://www.gov.uk/universal-credit and https://ucpostcode.entitledto.co.uk/ucdate to find out if you need to claim UC or ESA.  In both cases the questions and scoring on this site will apply</p>
```
Comment (katjam) [inline src/assessment.handlebars]: There might be an extra space before the final sentence and it is missing a fullstop at the end.

**Review by katjam — CHANGES_REQUESTED** [review]
Tiny changes.

File: src/assessment.handlebars (hunk 3 above)
Comment (katjam) [reply src/assessment.handlebars]: Should the urls be wrapped in <a>?

**Comment by harryharrold** [comment]
Two commits in response - sorry. Thank you for the review....

File: src/assessment.handlebars [hunk 4]
```diff
@@ -991,10 +992,11 @@
       <p><strong>Income-related ESA</strong> is paid to people whose income (looking at their income and their partners&#39;) and level of savings is low enough. What you receive is decided by the Government working out how much you need to live on, and paying you ESA to make up your income to that amount. In some circumstances income-related ESA may also be used to top up your contribution-based ESA.</p>
       <p>You cannot claim income-related ESA if you have savings of over &#xA3;16,000, or your partner works more than 24 hours per week.</p>
       <p>You can claim income-related ESA without time limit, although you may of course have your entitlement to ESA reassessed at some point.</p>
+      <p>In some parts of the country you can no longer make a new claim for income- related ESA, instead you claim Universal Credit. Eventually, Universal Credit will replace income-related ESA everywhere. This guide does not provide details about Universal Credit which you usually have to claim on-line. One of the places you can find out more <a href="https://www.gov.uk/universal-credit">information about Universal Credit is this page on gov.uk</a>.</p>
```
Comment (katjam) [inline src/assessment.handlebars]: Still an extra space.

**Review by katjam — CHANGES_REQUESTED** [review]

**Comment by harryharrold** [comment]
...did I get it right this time?

**Review by katjam — APPROVED** [review]
Yay! :koala:

**Pull request merged**